src/Utilities/StdHelpers.hpp

```diff
@@ -283,3 +283,74 @@ std::string formatted_string(const std::string& fmt, Args... args) {
   }
   return rtn;
 }
+
+// Arithmetic operators for std::array<T, Dim>
+
+template <size_t Dim, typename T>
+inline void operator+=(
```

Comment:
The operator X= traditionally return references to the result.  We may want to discourage the use of that feature in our code, but I don't know if some std stuff might rely on it.